tests/unit/django_spanner/test_compiler.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+from django.test import SimpleTestCase
+from django.core.exceptions import EmptyResultSet
+from django.db.utils import DatabaseError
+from django_spanner.compiler import SQLCompiler
+from django.db.models.query import QuerySet
+from .models import Number
+
+
+class TestUtils(SimpleTestCase):
```

Comment:
How come these classes are named `TestUtils` instead of `TestCompiler`, `TestExpressions`, etc?

Reply:
Copy paste, my bad, cleaned it up.